Translate nmt_with_attention.ipynb into Chinese

littlebeanbean7 opened the pull request:
Translate https://github.com/tensorflow/docs/blob/master/site/en/tutorials/text/nmt_with_attention.ipynb
into Chinese.

gaoljhy commented on site/zh-cn/tutorials/text/nmt_with_attention.ipynb:
```diff
@@ -0,0 +1,1036 @@
+    "\n",
+    "训练完此笔记本中的模型后，你将能够输入一个西班牙语句子，例如 *\"¿todavia estan en casa?\"*，并返回其英语翻译 *\"are you still at home?\"*\n",
+    "\n",
+    "翻译质量对于一个简单的例子来说是令人满意的，而更有趣的可能是生成的注意力图。它显示在翻译过程中，输入句子中的哪些部分抓住了模型的注意力。\n",
```
翻译质量对于一个简单的例子来说是令人满意的，而更有趣的可能是生成的注意力图。它显示在翻译过程中，输入句子中的哪些部分抓住了模型的注意力
不太通顺，烦请校正，谢谢

littlebeanbean7 replied:
Hi @gailjhy,

Many thanks for reviewing my work again. I rephrased a bit. Please kindly see whether it is what you refer to, otherwise please specify which phrase does not look good to you.

Thank you!
Cheers!

gaoljhy replied:
okay ,get it

tigerneil commented:
hidden size 直接译成"隐藏大小"不太合适，建议翻成 “隐藏层大小” @littlebeanbean7

littlebeanbean7 commented:
> hidden size 直接译成"隐藏大小"不太合适，建议翻成 “隐藏层大小” @littlebeanbean7

Hi XiaoHu,

Thanks for reviewing!
You are right, I changed this word throughout, also changed 隐藏状态 to 隐藏层状态.
Please kindly see whether it is appropriate now.

Best,
